README.md

```diff
@@ -129,6 +129,38 @@ example:
         },
     })
 
+### Using the parse_input method
+
+This method is used to validate incoming data against a pydantic model. A
+custom return type can be specifed in the case of validation failure, but
+it must extend flask_container_scaffold.BaseApiModel, or minimally implement a
+field 'error' or type string, so that the parse_input method can properly
```

Comment:
question: is this meant to say "of type string"?

Reply:
Yes, I'll fix that